playground-remote-inference.md

```diff
@@ -0,0 +1,72 @@
+# Remote inference in Playground
+
+Users can chat with external language model in playground.
```

Comment:
consider make it clear than explaining what's 'external model' --> Users can chat with remotely hosted language model in playground.

Reply:
Update